### PR TITLE
Independent website logo for company

### PR DESCRIPTION
**Pull request by rop-odoo**
Task: https://www.odoo.com/web#id=1968099&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720

### CODE REVIEW

File: odoo/addons/base/models/res_company.py [hunk 1]
```diff
@@ -72,6 +72,8 @@ def _get_default_favicon(self, original=False):
     # logo_web: do not store in attachments, since the image is retrieved in SQL for
     # performance reasons (see addons/web/controllers/main.py, Binary.company_logo)
     logo_web = fields.Binary(compute='_compute_logo_web', store=True, attachment=False)
+    report_display_logo = fields.Binary(string="Report Logo", compute='_compute_report_display_logo', inverse='_inverse_report_display_logo')
+    report_logo = fields.Binary(string="Report Logo Store")
```
Comment (qsm-odoo) [inline odoo/addons/base/models/res_company.py]: "Report Logo Store" means a shop which sells report logos... :) Remove the string definition and replace the one of the "report_display_logo" field by "Displayed Logo For Reports".

File: odoo/addons/base/models/res_company.py [hunk 2]
```diff
@@ -72,6 +72,8 @@ def _get_default_favicon(self, original=False):
     # logo_web: do not store in attachments, since the image is retrieved in SQL for
     # performance reasons (see addons/web/controllers/main.py, Binary.company_logo)
     logo_web = fields.Binary(compute='_compute_logo_web', store=True, attachment=False)
+    report_display_logo = fields.Binary(string="Report Logo", compute='_compute_report_display_logo', inverse='_inverse_report_display_logo')
```
Comment (qsm-odoo) [inline odoo/addons/base/models/res_company.py]: @JKE-be maybe we should move the compute and inverse method definition on top as you suggested ?

File: addons/website/models/website.py [hunk 3]
```diff
@@ -116,6 +121,7 @@ def _default_favicon(self):
         ('b2b', 'On invitation'),
         ('b2c', 'Free sign up'),
     ], string='Customer Account', default='b2b')
+    logo = fields.Binary(default=_default_logo)
```
Comment (qsm-odoo) [inline addons/website/models/website.py]: Move the field definition on top of the social links fields, for consistency.

File: addons/website/views/res_config_settings_views.xml [hunk 4]
```diff
@@ -130,6 +130,15 @@
                                     </div>
                                 </div>
                             </div>
+                            <div class="col-12 col-lg-6 o_setting_box" id="website_logo_setting">
+                                <div class="o_setting_right_pane">
+                                    <label for="website_logo"/>
```
Comment (rdeodoo) [inline addons/website/views/res_config_settings_views.xml]: Missing website specific icon:
```html
<span class="fa fa-lg fa-globe" title="Values set here are website-specific." groups="website.group_multi_website"/>
```

**Comment by rdeodoo** [comment]
@JKE-be @qsm-odoo This can be r+, every remarks have been processed

File: addons/website/models/website.py [hunk 5]
```diff
@@ -80,6 +80,12 @@ def _default_social_instagram(self):
     def _default_social_twitter(self):
         return self.env.ref('base.main_company').social_twitter
 
+    def _default_logo(self):
+        image_path = get_resource_path('website', 'static/src/img', 'res_website_logo.png')
```
Comment (JKE-be) [inline addons/website/models/website.py]: .

Comment (qsm-odoo) [reply addons/website/models/website.py]: @JKE-be ?

**Comment by JKE-be** [comment]
Could we rename img without res_?

Sorry, was just a reminder for @rde

Jeremy Kersten

**Comment by rdeodoo** [comment]
Done

**Comment by JKE-be** [comment]
@robodoo  r+

**Pull request closed without merging**